test/e2e/cypress/integration/profiles/recovery/return-to/success.spec.ts

```diff
@@ -0,0 +1,129 @@
+// Copyright © 2023 Ory Corp
+// SPDX-License-Identifier: Apache-2.0
+
+import { Identity } from "@ory/kratos-client"
+import { authenticator } from "otplib"
+import { gen, website } from "../../../../helpers"
+import { appPrefix, assertRecoveryAddress, gen } from "../../../../helpers"
+import { routes as express } from "../../../../helpers/express"
+import { routes as react } from "../../../../helpers/react"
+
+context("Recovery with `return_to`", () => {
+  ;[
+    //{
+    //  recovery: react.recovery,
+    //  settings: react.settings,
+    //  base: react.base,
+    //  app: "react" as "react",
+    //  profile: "spa",
+    //},
```

Comment:
What about the SPAs?

Reply:
Forgot to re-enable it for spa. But this will most likely need to be implemented in SPAs (from the spa side). The payload will return a Json with a redirect url. It remains up to the client how they want to deal with it.